src/Microsoft.AspNetCore.Mvc.Core/Internal/PrefixContainer.cs

```diff
@@ -25,8 +28,17 @@ public PrefixContainer(ICollection<string> values)
             }
 
             _originalValues = values;
-            _sortedValues = ToArrayWithoutNulls(_originalValues);
```

Comment:
To confirm: None of `IFormCollection`, `IQueryCollection` or `RouteValueDictionary` allow `null` keys? I had a quick look at the relevant code and can't find an implementation not backed by a real `Dictionary<K,V>`. But this method must have been added for reasons -- right...